doc/comparison-with-diyhue.md

```diff
@@ -22,7 +22,7 @@ DiyHue bridge, you will notice that it does not react to any changes from other
 phones, home automation, etc. Also, the reported light states (on/off, color,
 temperature, etc) are sometimes just wrong.
 
-Overall, DieHue can do an impressive number of things, but it seems to have some
+Overall, DiyHue can do an impressive number of things, but it seems to have some
```

Comment:
Oh damn, this one is particularly embarassing!

@hendriksen-mark, sorry! :sweat_smile: